filesystem/azure_test.go

```diff
@@ -37,7 +38,24 @@ func init() {
 	blobName = "textforexamination"
 }
 
+type nodeIO struct {
+	inputString []string
+}
+
+var flagtests = []nodeIO{
```

Comment:
Two points to correct:
1. nodeIO struct isn't needed. Just put
```
[]struct{
... your structs
}{
... your testing things
}
```
2. Put `flagtests` into the testing method as local variable.